code/game/gamemodes/clock_cult/clock_items/clockwork_slab.dm

```diff
@@ -342,16 +342,37 @@
 //- Components
 //- Scripture
 //- Power
-//- Lore
+//- Conversion
 /obj/item/clockwork/slab/proc/get_recollection_text(section)
 	var/list/dat = list()
 	switch(section)
 		if("Default")
 			dat += "You can browse the above sections as you please. They're designed to be read in order, but feel free to pick and choose between them."
+		if("Getting Started")
+			dat += "<font color=#BE8700 size=3>Getting Started</font><br><br>"
+			dat += "Welcome, servant! This section houses the utmost basics of being a servant of Ratvar, and is much more informal than the other sections. Being a servant of \
+			Ratvar is a very complex role, with many systems, objects, and resources to use effectively and creatively.<br><br>"
+			dat += "This section of your clockwork slab covers everything that servants have to be aware of, but is a long read because of how in-depth the systems are. Knowing \
+			how to use the tools at your disposal makes all the difference between a clueless servant and a great one.<br><br>"
+			dat += "If this is your first time being a servant, relax. It's very much possible that you'll fail, but it's impossible to learn without making mistakes. For the time \
+			being, use the Hierophant Network button in the top left-hand corner of your screen to try and get in touch with your fellow servants; ignore the others for now. This button \
+			will let you send messages across space and time to any other servant in the game world. This makes it great for coordinating, and you should use it often! <i>Note:</i> Using \
+			this will cause you to whisper your message aloud, so doing so in a public place is very suspicious and you should try to restrict it to private use.<br><br>"
+			dat += "If you aren't willing or don't have the time to read through every section, you can still help your teammates! Ask if they've set up a base. If they have, head there \
+			and ask however you can help; chances are there's always something. If not, it's your job as a servant to get one up and running! Try to find a secluded, low-traffic area, \
+			like the auxilary base or somewhere deep in maintenance. You'll want to go into the Drivers section of the slab and look for <i>Tinkerer's Cache.</i> Find a nice spot and \
+			create one. This serves as a storage for <i>components,</i> the cult's primary resource. (Your slab's probably produced a few by now.) By attaching that cache with this \
```

Comment:
you mean you're _not_ supposed to tape a cache to your slab? guess I've been playing wrong.